src/otx/algo/object_detection_3d/losses/ddn_loss.py

```diff
@@ -76,12 +76,11 @@ def forward(
         """Forward pass.
 
         Args:
-            loss [torch.Tensor(B, H, W)]: Pixel-wise loss
-            gt_boxes2d [torch.Tensor (B, N, 4)]: 2D box labels for foreground/background balancing
+            loss (torch.Tensor(B, H, W)): Pixel-wise loss
+            gt_boxes2d (torch.Tensor (B, N, 4)): 2D box labels for foreground/background balancing
 
         Returns:
-            loss [torch.Tensor(1)]: Total loss after foreground/background balancing
-            tb_dict [dict[float]]: All losses to log in tensorboard
+            loss (torch.Tensor(1)): Total loss after foreground/background balancing
```

Comment:
Is it normal to insert shape in type hint?

Reply:
Done https://github.com/openvinotoolkit/training_extensions/pull/4031/commits/8c2576f094dc7630e6d26333cddebb785d42d76f